webui/react/src/utils/number.ts

```diff
@@ -2,6 +2,21 @@ import { isString } from './data';
 
 const PERCENT_REGEX = /^(\d+\.?\d*|\.\d+)%$/;
 
+export const findFactorOfNumber = (n: number): number[] => {
+  const abs = Math.abs(n);
+  const factors = [];
+
+  for (let i = 1; i <= Math.floor(Math.sqrt(abs)); i++) {
+    if (abs % i !== 0) continue;
+    factors.push(i);
+
+    if (abs/i === i) continue;
```

Comment:
shouldn't need this check because if `i` is divisible `abs/i` should also be.

Reply:
Are you 100% sure? I'm not and not really sure how to verify.